content/en/apps/_index.md

```diff
@@ -4,20 +4,13 @@ linkTitle: "CHT Applications"
 identifier: "apps"
 weight: 2
 description: >
-  Overview and reference for building digital health apps
+  Overview and reference for building CHT applications
 ---
 
-{{% pageinfo %}}
-This section provides an overview and reference for building digital health applications with CHT Core.
-{{% /pageinfo %}}
+To create a digital health tool using the Community Health Toolkit: build a CHT Application or re-use an existing CHT Application. The [CHT Core Framework]({{ ref="core" }}) provides a foundation on which many digital health tools can be conceptualised and built. A CHT Application takes that foundation and builds on it to create a concrete digital health tool tailored to the health workflows in a targeted community. 
```

Comment:
I find this paragraph tough to read and get into. We also remove the concept of configuration code, which is something we've used organizationally for a while. I am ok with changing the term, but it is useful as a concept so it would be good to have a larger conversation about what we want to call that.

An attempt to reword, feel free to use as you wish.

```suggestion
CHT applications are digital health apps built using the Core Framework of the Community Health Toolkit. The [CHT Core Framework]({{< ref="core" >}}) provides a foundation on which a concrete digital health app can be built, tailored to the needs of the health workers and the community. This custom building of CHT apps is done in a modular way, bringing in settings, images, and code as needed.
```

Minor note that from what I've seen we typically use "app" over "tool", unless it is a specific aspect within the app. @CescaGarrett might have more on that language.

Reply:
I thought we renamed "configuration code" as "application code"?

Reply:
I wasn't aware (or forgot) about that change... I remember the cons of "applicaton code" were that it makes it hard to distinguish with what was previously _non-configuration code_, and I don't remember us renaming that to _framework code_, _core application code_, or something else.